kernel/examples/run_examples.sh

```diff
@@ -0,0 +1,49 @@
+##
+## This script runs the Delta Kernel example programs using the golden
+## tables located in the <repo-root>/connectors/golden-tables/src/main/resources/golden
+## directory.
+##
+## Make sure to run this script from <repo-root> in order for the relative
```

Comment:
why not make this script be runnable from anywhere?

Reply:
also .. thats why writing such scripts in python gives a lot more flexibility and an entire library of tools to do these sort of stuff. There is a lot of python scripts in this repo (e.g., run-integration-tests.py) that you can steal from.

Reply:
Rewrote this in python (by copying the code from delta integration test).